actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -88,7 +88,14 @@ def translate(key, options = {})
           raise e if raise_error
 
           keys = I18n.normalize_keys(e.locale, e.key, e.options[:scope])
-          content_tag('span', keys.last.to_s.titleize, :class => 'translation_missing', :title => "translation missing: #{keys.join('.')}")
+          title = "translation missing: #{keys.join('.')}"
+
+          interpolation = options.except(:default)
```

Comment:
`except` doesn't mutate the `options`:

```
irb(main):001:0> options = { default: 'ahoy' }
=> {:default=>"ahoy"}
irb(main):002:0> options.except(:default)
=> {}
irb(main):003:0> options
=> {:default=>"ahoy"}
```

So back to my earlier question of why we're using `options` below and not `interpolations`?

(By the way the `interpolations` variable aught to be plural).